plugins/core/jetbrains-community/resources/telemetryOverride.json

```diff
@@ -44,6 +44,44 @@
             ],
             "description": "Identifies the specific interaction that opens the chat panel"
         },
+        {
+            "name": "codewhispererLanguage",
+            "type": "string",
+            "description": "Programming language of the CodeWhisperer recommendation",
```

Comment:
any chance for not using the override?

Reply:
yea i will remove it.

Reply:
working on telemetry version bump

Reply:
well, have to do it in a separate PR #4900 as there is gumby changes needed to be made